### PR TITLE
FM-243, FM-246 - Reusable Helpers functions and Aliasing

### PR DESCRIPTION
**Pull request by woodsjohn-codev**

### CODE REVIEW

**Comment by bernhardccodev** [comment]
@woodsjohn-codev What file changes for FM-243 Create a Reusable Helpers File in FTM?
I can only see FM-246 changes for aliasing.

File: src/interfaces/StoneHandlerInterface.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { StoneConfig } from "../common";
+import { AudioPlayer, TimerTicking, Tutorial } from "../components";
```
Comment (bernhardccodev) [inline src/interfaces/StoneHandlerInterface.ts:2]: Utilize aliasing and index

File: src/interfaces/feedbackTextEffectsInterface.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { TextParticle } from "@components/feedback-particle-effect/text-particle";
```
Comment (bernhardccodev) [inline src/interfaces/feedbackTextEffectsInterface.ts:1]: Utilize index

File: src/interfaces/levelEndSceneInterface.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { Monster, Background, AudioPlayer } from "../components";
+import CloseButton from "@components/buttons/close-button";
+import NextButton from "@components/buttons/next-button";
+import RetryButton from "@components/buttons/retry-button";
```
Comment (bernhardccodev) [inline src/interfaces/levelEndSceneInterface.ts:4]: Utilize alias and index

File: src/sceneHandler/scene-handler.ts [hunk 4]
```diff
@@ -38,10 +38,7 @@ export class SceneHandler {
   private pwa_install_status: Event;
   private toggleBtn: HTMLElement;
```
Comment (bernhardccodev) [inline src/sceneHandler/scene-handler.ts:39]: Same question for interface

**Review by bernhardccodev — COMMENTED** [review]
@woodsjohn-codev please see comments

File: src/scenes/gameplay-scene.ts [hunk 5]
```diff
@@ -189,6 +196,10 @@ export class GameplayScene {
     this.feedBackTexts = feedBackTexts;
 
     this.images = {
+      pillerImg: ASSETS_PATH_TOTEM,
+      bgImg: ASSETS_PATH_BG_01,
+      hillImg: ASSETS_PATH_HILL,
+      fenchImg: ASSETS_PATH_FENCE,
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/scenes/gameplay-scene.ts]: @bernhardccodev I fixed this on my latest commit. thanks

File: src/common/stone-config.ts [hunk 6]
```diff
@@ -1,113 +1,117 @@
 import { font, lang } from "../../global-variables";
-import { TimerTicking } from "../components/timer-ticking";
-import { Tutorial } from "../components/tutorial";
-import { Utils } from "./utils";
+import { TimerTicking } from "@components/timer-ticking";
+import { Tutorial } from "@components/tutorial";
+import { drawImageOnCanvas } from "@common/index";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/common/stone-config.ts]: @bernhardccodev I fixed this on my latest commit. thank you

File: src/common/stone-config.ts [hunk 7]
```diff
@@ -1,113 +1,117 @@
 import { font, lang } from "../../global-variables";
-import { TimerTicking } from "../components/timer-ticking";
-import { Tutorial } from "../components/tutorial";
-import { Utils } from "./utils";
+import { TimerTicking } from "@components/timer-ticking";
+import { Tutorial } from "@components/tutorial";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/common/stone-config.ts:3]: @bernhardccodev I fixed this on my latest commit. thank you

File: src/common/utils.ts [hunk 8]
```diff
@@ -1,5 +1,6 @@
 import { Debugger, lang } from "../../global-variables";
-import languageFontMapping from "../data/i18-font-mapping";
+import { AudioPlayer } from "../components";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/common/utils.ts]: @bernhardccodev I fixed this on my latest commit. thank you

File: src/scenes/level-selection-scene.ts [hunk 9]
```diff
@@ -374,36 +419,26 @@ export class LevelSelectionScreen {
...
-      );
+    const positions = [
+      [0, -imageSize * 0.01],
+      [imageSize / 2.5, -imageSize * 0.01],
+      [imageSize / 5, -imageSize * 0.1],
+    ];
+
+    for (let i = 0; i < starCount; i++) {
+      const [dx, dy] = positions[i];
+      this.drawImage("star", s.x + dx, s.y + dy, imageSize / 5, imageSize / 5);
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/scenes/level-selection-scene.ts:455]: its on https://github.com/curiouslearning/FeedTheMonsterJS/blob/51cc34d478c8a244e0ec5be63f1f25470b5fa769/src/scenes/level-selection-scene.ts#L295C11-L295C20

File: src/scenes/levelend-scene.ts [hunk 10]
```diff
@@ -1,25 +1,19 @@
...
-  AUTUMN_BG_GROUP_IMGS,
-  WINTER_BG_GROUP_IMGS
-} from '../constants';
+import { loadImages } from "@common/index";
+import { CLICK } from "@common/event-names";
+import { AudioPlayer } from "@components/audio-player";
+import CloseButton from "@components/buttons/close-button";
+import NextButton from "@components/buttons/next-button";
+import RetryButton from "@components/buttons/retry-button";
+import { Monster } from "@components/monster";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/scenes/levelend-scene.ts:7]: @bernhardccodev thanks i fixed on my latest commit.

File: src/scenes/levelend-scene.ts [hunk 11]
```diff
@@ -1,25 +1,19 @@
...
   BACKGROUND_ASSET_LIST,
   createBackground,
   loadDynamicBgAssets,
-  defaultBgDrawing,
-  autumBgDrawing,
-  winterBgDrawing
-} from '../compositions/background';
+} from "../compositions/background";
+import { drawImageOnCanvas } from "@common/index";
+import { LevelEndSceneInterface } from "@interfaces/levelEndSceneInterface";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/scenes/levelend-scene.ts:14]: @bernhardccodev thanks i fixed on my latest commit.

File: src/scenes/start-scene.ts [hunk 12]
```diff
@@ -1,19 +1,20 @@
...
+import PlayButton from "@components/buttons/play-button";
+import { createBackground, defaultBgDrawing } from "../compositions/background";
 import {
   FirebaseUserClicked,
   PWAInstallStatus,
-  DEFAULT_BG_GROUP_IMGS
-} from '../constants/';
+  DEFAULT_BG_GROUP_IMGS,
+} from "../constants/";
+import { StartSceneInterface } from "@interfaces/startSceneInterface";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/scenes/start-scene.ts:15]: @bernhardccodev thanks i fixed on my latest commit.

File: src/components/buttons/cancel-button.ts [hunk 13]
```diff
@@ -1,49 +1,53 @@
...
-    public posY: number;
-    public context: CanvasRenderingContext2D;
-    public canvas: { width: any; height?: number };
-    public imagesLoaded: boolean = false;
-    public cancel_button_image: HTMLImageElement;
+import {
+  drawImageOnCanvas,
+  isClickInsideButton,
+  loadImages,
+} from "@common/index";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/components/buttons/cancel-button.ts]: @bernhardccodev thanks i fixed on my latest commit.

File: src/components/buttons/cancel-button.ts [hunk 14]
```diff
@@ -1,49 +1,53 @@
...
-    }
+    loadImages({ button_image: "./assets/images/close_btn.png" }, (images) => {
+      this.button_image = images["button_image"];
+      this.imagesLoaded = true;
+    });
+  }
 
-    onClick(xClick: number, yClick: number): boolean {
-        const distance = Math.sqrt(
-            (xClick - this.posX - (this.canvas.width * 0.15) / 2) *
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/components/buttons/cancel-button.ts:40]: I set a reusable code here on utils https://github.com/curiouslearning/FeedTheMonsterJS/blob/FM-243/src/common/utils.ts#L180-L193

File: src/components/buttons/cancel-button.ts [hunk 15]
```diff
@@ -1,49 +1,53 @@
...
-    public context: CanvasRenderingContext2D;
-    public canvas: { width: any; height?: number };
-    public imagesLoaded: boolean = false;
-    public cancel_button_image: HTMLImageElement;
+import {
+  drawImageOnCanvas,
+  isClickInsideButton,
+  loadImages,
+} from "@common/index";
+import { ButtonInterface } from "@interfaces/buttons";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/components/buttons/cancel-button.ts:2]: @bernhardccodev thanks i fixed on my latest commit.

File: src/components/buttons/close-button.ts [hunk 16]
```diff
@@ -1,49 +1,53 @@
+import {
+  drawImageOnCanvas,
+  isClickInsideButton,
+  loadImages,
+} from "@common/index";
+import { ButtonInterface } from "@interfaces/buttons";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/components/buttons/close-button.ts:2]: @bernhardccodev thanks i fixed on my latest commit.

File: src/components/buttons/next-button.ts [hunk 17]
```diff
@@ -1,52 +1,58 @@
...
-    public width: number;
-    public height: number;
-    public imagesLoaded: boolean = false;
-    public next_button_image: HTMLImageElement;
+import {
+  drawImageOnCanvas,
+  isClickInsideButton,
+  loadImages,
+} from "@common/index";
+import { ButtonInterface } from "@interfaces/buttons";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/components/buttons/next-button.ts:2]: @bernhardccodev thanks i fixed on my latest commit.

File: src/components/buttons/no-button.ts [hunk 18]
```diff
@@ -1,10 +1,25 @@
...
-  public posY: number;
-  public context: CanvasRenderingContext2D;
-  public canvas: { width: any; height?: number };
-  public imagesLoaded: boolean = false;
-  public no_button_image: HTMLImageElement;
+import {
+  drawImageOnCanvas,
+  isClickInsideButton,
+  loadImages,
+} from "@common/index";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/components/buttons/no-button.ts]: @bernhardccodev thanks i fixed on my latest commit.

File: src/components/buttons/play-button.ts [hunk 19]
```diff
@@ -0,0 +1,94 @@
+import { TappedStart } from "../../Firebase/firebase-event-interface";
+import { loadImages } from "../../common";
+import { FirebaseIntegration } from "../../Firebase/firebase-integration";
+import { lang, pseudoId } from "../../../global-variables";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/components/buttons/play-button.ts]: @bernhardccodev thanks i fixed on my latest commit.

File: src/components/buttons/play-button.ts [hunk 20]
```diff
@@ -0,0 +1,94 @@
...
+import { getData } from "@data/api-data";
+import { drawImageOnCanvas } from "@common/index";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/components/buttons/play-button.ts]: @bernhardccodev thanks i fixed on my latest commit.

File: src/components/buttons/retry-button.ts [hunk 21]
```diff
@@ -1,53 +1,63 @@
+import {
+  drawImageOnCanvas,
+  isClickInsideButton,
+  loadImages,
+} from "@common/index";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/components/buttons/retry-button.ts]: @bernhardccodev thanks i fixed on my latest commit.

File: src/components/buttons/yes-button.ts [hunk 22]
```diff
@@ -1,10 +1,25 @@
...
-  public posY: number;
-  public context: CanvasRenderingContext2D;
-  public canvas: { width: any; height?: number };
-  public imagesLoaded: boolean = false;
-  public yes_button_image: HTMLImageElement;
+import {
+  drawImageOnCanvas,
+  isClickInsideButton,
+  loadImages,
+} from "@common/index";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/components/buttons/yes-button.ts]: @bernhardccodev thanks i fixed on my latest commit.

File: src/components/feedback-particle-effect/feedback-text-effects.ts [hunk 23]
```diff
@@ -1,7 +1,8 @@
 import { TextParticle } from "./text-particle";
 import { font } from "../../../global-variables";
+import { FeedbackTextEffectsInterface } from "@interfaces/feedbackTextEffectsInterface";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/components/feedback-particle-effect/feedback-text-effects.ts:3]: @bernhardccodev thanks i fixed on my latest commit.

File: src/common/stone-config.ts (hunk 7 above)
[the remark this answers is not in the document]
Comment (bernhardccodev) [reply src/common/stone-config.ts:3]: Don't unresolved this. I will resolved this once I have seen the changes.

File: src/components/buttons/no-button.ts [hunk 24]
```diff
@@ -1,10 +1,25 @@
...
+import {
+  ButtonImage,
+  ButtonInterface,
+  Canvas,
+  ImagesLoaded,
+  PosX,
+  PosY,
+  Context,
+} from "@interfaces/buttons";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/components/buttons/no-button.ts:10]: @bernhardccodev thanks i fixed on my latest commit.

File: src/components/buttons/pause-button.ts [hunk 25]
```diff
@@ -1,47 +1,56 @@
...
+} from "@common/index";
+import {
+  ButtonImage,
+  ButtonInterface,
+  Canvas,
+  Context,
+  ImagesLoaded,
+  PosX,
+  PosY,
+} from "@interfaces/buttons";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/components/buttons/pause-button.ts:10]: @bernhardccodev thanks i fixed on my latest commit.

File: src/components/buttons/retry-button.ts [hunk 26]
```diff
@@ -1,53 +1,63 @@
...
+import {
+  ButtonImage,
+  ButtonInterface,
+  Canvas,
+  Context,
+  ImagesLoaded,
+  PosX,
+  PosY,
+} from "@interfaces/buttons";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/components/buttons/retry-button.ts:10]: @bernhardccodev thanks i fixed on my latest commit.

File: src/components/buttons/yes-button.ts [hunk 27]
```diff
@@ -1,10 +1,25 @@
...
+import {
+  ButtonImage,
+  ButtonInterface,
+  Canvas,
+  Context,
+  ImagesLoaded,
+  PosX,
+  PosY,
+} from "@interfaces/buttons";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/components/buttons/yes-button.ts:10]: @bernhardccodev thanks i fixed on my latest commit.

File: src/components/feedback-particle-effect/feedback-text-effects.ts [hunk 28]
```diff
@@ -141,7 +153,10 @@ export class FeedbackTextEffects {
   }
 
   public unregisterEventListener(): void {
-    this.textWorker.removeEventListener("message", this.handleTextWorkerMessage);
+    this.textWorker.removeEventListener(
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/components/feedback-particle-effect/feedback-text-effects.ts]: @bernhardccodev updated the disposeEventListeners to apply this code

File: src/components/feedback-particle-effect/sure-popup.ts [hunk 29]
```diff
@@ -1,7 +1,8 @@
-import { CLICK } from "../../common/event-names";
+import { CLICK } from "@common/event-names";
 import { AudioPlayer } from "../audio-player";
 import YesButton from "../buttons/yes-button";
 import NoButton from "../buttons/no-button";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/components/feedback-particle-effect/sure-popup.ts]: @bernhardccodev thanks i fixed on my latest commit.

File: src/components/level-indicator.ts [hunk 30]
```diff
@@ -1,85 +1,90 @@
 import { loadImages } from "../common";
+import { drawImageOnCanvas } from "@common/index";
 import { EventManager } from "../events/EventManager";
+import { LevelIndicatorsInterface } from "@interfaces/levelIndicatorsInterface";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/components/level-indicator.ts:4]: @bernhardccodev thanks i fixed on my latest commit.

File: src/components/feedback-particle-effect/sure-popup.ts [hunk 31]
```diff
@@ -1,7 +1,8 @@
...
+import { drawImageOnCanvas, loadImages } from "@common/index";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/components/feedback-particle-effect/sure-popup.ts]: @bernhardccodev thanks i fixed on my latest commit.

File: src/events/EventManager.ts [hunk 32]
```diff
@@ -1,4 +1,4 @@
-import { LOADPUZZLE, STONEDROP } from "../common/event-names";
+import { LOADPUZZLE, STONEDROP } from "@common/event-names";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/events/EventManager.ts:1]: @bernhardccodev thanks i fixed on my latest commit.

File: src/interfaces/StoneHandlerInterface.ts (hunk 1 above)
Comment (woodsjohn-codev) [reply src/interfaces/StoneHandlerInterface.ts]: 1. @bernhardccodev thanks i fixed on my latest commit.

File: src/interfaces/feedbackTextEffectsInterface.ts (hunk 2 above)
Comment (woodsjohn-codev) [reply src/interfaces/feedbackTextEffectsInterface.ts:1]: @bernhardccodev thanks i fixed on my latest commit.

File: src/interfaces/gameplaySceneInterface.ts [hunk 33]
```diff
@@ -0,0 +1,94 @@
+import { StoneConfig } from "../common";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/interfaces/gameplaySceneInterface.ts]: @bernhardccodev thanks i fixed on my latest commit.

File: src/interfaces/gameplaySceneInterface.ts [hunk 34]
```diff
@@ -0,0 +1,94 @@
...
+  LevelIndicators,
+  Monster,
+  PauseButton,
+  PausePopUp,
+  PromptText,
+  StoneHandler,
+  TimerTicking,
+  Tutorial,
+} from "../components";
+import { FirebaseIntegration } from "../Firebase/firebase-integration";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/interfaces/gameplaySceneInterface.ts:15]: @bernhardccodev thanks i fixed on my latest commit.

File: src/interfaces/levelEndSceneInterface.ts (hunk 3 above)
Comment (woodsjohn-codev) [reply src/interfaces/levelEndSceneInterface.ts:4]: @bernhardccodev thanks i fixed on my latest commit.

File: src/components/monster.ts [hunk 35]
```diff
@@ -1,10 +1,9 @@
-import { Debugger, lang } from "../../global-variables";
-import { StoreMonsterPhaseNumber } from '../constants/'
 import { loadImages } from "../common";
 import { EventManager } from "../events/EventManager";
+import { drawImageOnCanvas } from "@common/index";
+import { MonsterInterface } from "@interfaces/monsterInterface";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/components/monster.ts:4]: @bernhardccodev thanks i fixed on my latest commit.

File: src/components/monster.ts [hunk 36]
```diff
@@ -45,7 +44,7 @@ export class Monster extends EventManager {
     this.maxFrame = 6;
     this.x = this.game.width / 2 - this.game.width * 0.243;
     this.y = this.game.width / 3;
-    // console.log(this.x,this.y); 
+    // console.log(this.x,this.y);
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/components/monster.ts]: @bernhardccodev thanks i fixed on my latest commit.

File: src/components/pause-popup.ts [hunk 37]
```diff
@@ -1,10 +1,12 @@
...
-import RetryButton from "../components/buttons/retry-button";
-import { CLICK, VISIBILITY_CHANGE } from "../common/event-names";
+import CancelButton from "@components/buttons/cancel-button";
+import CloseButton from "@components/buttons/close-button";
+import RetryButton from "@components/buttons/retry-button";
+import { CLICK } from "@common/event-names";
 import { AudioPlayer } from "./audio-player";
 import AreYouSurePopUp from "./feedback-particle-effect/sure-popup";
 import { lang } from "../../global-variables";
+import { drawImageOnCanvas, loadImages } from "@common/index";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/components/pause-popup.ts]: @bernhardccodev thanks i fixed on my latest commit.

File: src/components/timer-ticking.ts [hunk 38]
```diff
@@ -1,129 +1,144 @@
-import { loadImages } from "../common/";
+import { loadImages } from "@common/index";
+import { drawImageOnCanvas } from "@common/index";
 import { EventManager } from "../events/EventManager";
+import { TimerTickingInterface } from "@interfaces/timeTickingInterface";
 import { AudioPlayer } from "./audio-player";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/components/timer-ticking.ts:5]: @bernhardccodev thanks i fixed on my latest commit.

File: src/components/tutorial.ts [hunk 39]
```diff
@@ -1,5 +1,5 @@
-import { createRippleEffect } from "../common/utils";
-import { GameScore } from "../data/game-score";
+import { createRippleEffect, drawImageOnCanvas } from "@common/index";
+import { GameScore } from "@data/game-score";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/components/tutorial.ts:2]: @bernhardccodev thanks i fixed on my latest commit.

File: src/compositions/background/index.ts [hunk 40]
```diff
@@ -1,161 +1,169 @@
...
+import { drawImageOnCanvas } from "@common/index";
 import {
-    DEFAULT_BG_GROUP_IMGS,
-    AUTUMN_BG_GROUP_IMGS,
-    WINTER_BG_GROUP_IMGS
-} from '../../constants/';
+  DEFAULT_BG_GROUP_IMGS,
+  AUTUMN_BG_GROUP_IMGS,
+  WINTER_BG_GROUP_IMGS,
+} from "../../constants/";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/compositions/background/index.ts]: @bernhardccodev thanks i fixed on my latest commit.

File: src/interfaces/promptTextInterface.ts [hunk 41]
```diff
@@ -0,0 +1,44 @@
+import { AudioPlayer } from "../components";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/interfaces/promptTextInterface.ts]: @bernhardccodev thanks i fixed on my latest commit.

File: src/interfaces/sceneHandlerInterface.ts [hunk 42]
```diff
@@ -0,0 +1,37 @@
+import { StartScene } from "../scenes/start-scene";
+import { DataModal } from "@data/data-modal";
+import { LevelSelectionScreen } from "../scenes/level-selection-scene";
+import { GameplayScene } from "../scenes/gameplay-scene";
+import { LoadingScene } from "../scenes/loading-scene";
+import { LevelEndScene } from "../scenes/levelend-scene";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/interfaces/sceneHandlerInterface.ts]: @bernhardccodev thanks i fixed on my latest commit.

File: src/interfaces/startSceneInterface.ts [hunk 43]
```diff
@@ -0,0 +1,34 @@
+import { StoneConfig } from "../common";
+import { Monster, Background, AudioPlayer } from "../components";
+import PlayButton from "@components/buttons/play-button";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/interfaces/startSceneInterface.ts:3]: @bernhardccodev thanks i fixed on my latest commit.

File: src/interfaces/timeTickingInterface.ts [hunk 44]
```diff
@@ -0,0 +1,34 @@
+import { AudioPlayer } from "../components";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/interfaces/timeTickingInterface.ts]: @bernhardccodev thanks i fixed on my latest commit.

File: src/sceneHandler/scene-handler.ts [hunk 45]
```diff
@@ -16,10 +16,10 @@ import {
   LevelSelection1,
   GameScene1,
   EndScene1,
-} from '../constants';
+} from "../constants";
+import { SceneHandlerInterface } from "@interfaces/sceneHandlerInterface";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/sceneHandler/scene-handler.ts:20]: @bernhardccodev thanks i fixed on my latest commit.

File: src/scenes/gameplay-scene.ts [hunk 46]
```diff
@@ -24,31 +24,36 @@ import {
...
   createBackground,
   loadDynamicBgAssets,
   defaultBgDrawing,
   autumBgDrawing,
-  winterBgDrawing
-} from '../compositions/background';
+  winterBgDrawing,
+} from "../compositions/background";
+import { disposeEventListeners } from "@common/index";
+import { GameplaySceneInterface } from "@interfaces/gameplaySceneInterface";
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/scenes/gameplay-scene.ts:51]: @bernhardccodev thanks i fixed on my latest commit.

File: src/scenes/gameplay-scene.ts [hunk 47]
```diff
@@ -78,6 +83,10 @@ export class GameplayScene {
   stoneHandler: StoneHandler;
   public counter: number = 0;
   images: {
+    pillerImg: string;
+    bgImg: string;
+    hillImg: string;
+    fenchImg: string;
     profileMonster: string;
   };
   handler: HTMLElement;
```
[the remark this answers is not in the document]
Comment (woodsjohn-codev) [reply src/scenes/gameplay-scene.ts]: @bernhardccodev thanks i fixed on my latest commit.

**Pull request closed without merging**